build_openwebtext_pretraining_dataset.py

```diff
@@ -78,6 +79,11 @@ def main():
                       help="Number of tokens per example.")
   parser.add_argument("--num-processes", default=1, type=int,
                       help="Parallelize across multiple processes.")
+  parser.add_argument("--do-lower-case", dest='do_lower_case',
```

Comment:
no add_argument for blanks_separate?

Reply:
Thanks for that hint! I added it with the last commit and set it to `False` like it is used in the  OpenWebTextCorpus script.

Reply:
I actually don't think it is needed, as you should always have blanks-separate-docs=False for the OpenWebTextCorpus (each doc is a separate file and there is a blank between every sentence).

Reply:
No problem, I just reverted the last commit :)